setup.py

```diff
@@ -1,12 +1,16 @@
 from setuptools import setup
 
+#bring in __version__ from source
+#per https://stackoverflow.com/a/17626524
+#and https://stackoverflow.com/a/2073599
+with open('lineman/version.py') as ver: exec(ver.read())
```

Comment:
It seemed to be the most supported way to do it. The race condition was if, in setup.py, I do 'from lineman import __version__, like I do in __main__, there would be a problem on a new install since lineman hasn't been installed yet.